io.github.electronstudio.WeylusCommunityEdition.json

```diff
@@ -0,0 +1,65 @@
+    "org.freedesktop.Sdk.Extension.node18"
+  ],
+  "modules" : [ {
+    "name" : "WeylusCommunityEdition",
+    "buildsystem" : "simple",
+    "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin:/usr/lib/sdk/node18/bin",
+      "env": {
+        "RUSTFLAGS":"-A dead_code -A unused_imports -L /app/lib",
+        "CARGO_HOME": "/run/build/WeylusCommunityEdition/cargo",
+        "XDG_CACHE_HOME": "/run/build/WeylusCommunityEdition/flatpak-node/cache",
+        "npm_config_cache": "/run/build/WeylusCommunityEdition/flatpak-node/npm-cache",
+        "npm_config_nodedir": "/usr/lib/sdk/node18"
+      }
+    },
+    "build-commands" : [
+      "tar xvf x264-x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1.tar.bz2",
+      "cd x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1; ./configure --prefix=/app --exec-prefix=/app --disable-opencl --disable-cli --enable-static --enable-shared; make -j8; make install",
+      "tar xvf ffmpeg-6.1.1.tar.xz",
+      "cd ffmpeg-6.1.1 ; ./configure --prefix=/app --disable-debug --disable-programs --enable-gpl --enable-libx264  --disable-autodetect --enable-static --enable-shared; C_INCLUDE_PATH=/app/include LIBRARY_PATH=/app/lib make -j8; make install",
+      "sed -i 's/npm install/npm install --offline/g' build.rs",
+      "C_INCLUDE_PATH=/app/include LIBRARY_PATH=/app/lib cargo b --features ffmpeg-system --release",
+      "install -D target/release/weylus /app/bin/weylus",
+      "install -D weylus.desktop /app/share/applications/io.github.electronstudio.WeylusCommunityEdition.desktop",
+      "install -D io.github.electronstudio.WeylusCommunityEdition.metainfo.xml /app/share/metainfo/io.github.electronstudio.WeylusCommunityEdition.metainfo.xml",
+      "install -D io.github.electronstudio.WeylusCommunityEdition.svg /app/share/icons/hicolor/scalable/apps/io.github.electronstudio.WeylusCommunityEdition.svg"
+    ],
+    "sources" : [
+      {
+        "type": "archive",
+        "url": "https://github.com/electronstudio/WeylusCommunityEdition/archive/refs/tags/2024.7.31.tar.gz",
+        "sha512": "da10a17fd798805b2bee033019af25949d15f6777945e04c71e0208860bc72e421ec7a1a03821f039aefa619650dcab305cde55d4934938bd128d1803c21ec71"
+      },
+      {"type": "file",
+      "url": "https://code.videolan.org/videolan/x264/-/archive/4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1/x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1.tar.bz2",
+      "dest-filename": "x264-x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1.tar.bz2",
+      "sha512": "6d0d9e079d6c7650abb5c00ad60cf3cfe72a220b3ee7cd030d4daeefdd9feeb4d056cf2e01b2f8d2fb0a66ccc15cdde860237f8ac5eac42ede6e5444a81346f8"
+      },
+      {"type": "file",
+        "url": "https://www.ffmpeg.org/releases/ffmpeg-6.1.1.tar.xz",
+        "dest-filename": "ffmpeg-6.1.1.tar.xz",
+        "sha512": "fca3f8635f29182e3ae0fe843a8a53614e4b47e22c11508df3ff7cdbafbb4b5ee0d82d9b3332871f7c1032033b1cad2f67557d7c5f7f7d85e2adadca122965d5"
+      },
```

Comment:
ffmpge and x264 should be built in a separate module before this,

Reply:
Also remove the "file" sources, these should be archive sources.

Reply:
There are lots of examples on Flathub https://github.com/search?q=org%3Aflathub%20%22ffmpeg-6.1.1.tar.xz%22&type=code